listish.py

```diff
@@ -4,11 +4,23 @@
 """
 
 from collections import Sequence, MutableSequence
+from sys import version_info
+
+
+if version_info < (3, 0):  # pragma: no cover
```

Comment:
Could have done the `try: xrange; except NameError: xrange = range` trick here instead, but making Python 3 the special case seems backwards.